static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.spec.tsx

```diff
@@ -111,12 +119,20 @@ describe('SpanEvidenceKeyValueList', () => {
         screen.getByTestId('span-evidence-key-value-list.starting-span')
       ).toHaveTextContent('db - SELECT * FROM USERS LIMIT 100');
 
-      expect(
-        screen.queryByRole('cell', {name: 'Parallelizable Span'})
-      ).toBeInTheDocument();
-      expect(
-        screen.getByTestId('span-evidence-key-value-list.parallelizable-span')
-      ).toHaveTextContent('db - SELECT COUNT(*) FROM USERS');
+      expect(screen.queryAllByRole('cell', {name: 'Parallelizable Spans'}).length).toBe(
+        1
+      );
+      const parallelizableSpanKeyValue = screen.getByTestId(
+        'span-evidence-key-value-list.parallelizable-spans'
+      );
+
+      expect(parallelizableSpanKeyValue).toHaveTextContent(
+        'db - SELECT COUNT(*) FROM USERS'
+      );
+      expect(parallelizableSpanKeyValue).toHaveTextContent(
+        'db - SELECT COUNT(*) FROM ITEMS'
+      );
+      expect(parallelizableSpanKeyValue.getElementsByTagName('pre').length).toBe(2);
```

Comment:
```suggestion
```

Honestly, I think you can ditch this! This is more of a test of how `KeyValueList` renders multiple items (maybe a good test to have, actually!) than anything to do with span evidence. The `toHaveTextContent` stuff is great

Reply:
I was trying to test if "Parallelizable Spans" doesn't actually render twice (and therefore there are two 'pre' elements under one parallelizbale Span key. Would probably be better to just check if that text only appears once?
I'll remove this and save that for another PR.

Reply:
That makes sense! I more mean that this test really belong with `KeyValueStore` since that's where the logic for multi-values lives